plugins/inputs/lvm/lvm.go

```diff
@@ -284,6 +284,11 @@ type lvsReport struct {
 
 func init() {
 	inputs.Add("lvm", func() telegraf.Input {
-		return &LVM{}
+		return &LVM{
+			UseSudo:   false,
```

Comment:
Please remove this line as this is the Go default. Same in the test cases.

Reply:
Done in commit https://github.com/influxdata/telegraf/pull/12725/commits/a24a4faaf86e7f5060d50d4ff2e79f74fecee360